pkg/controller/nodesetup/sync_raids.go

```diff
@@ -117,7 +125,7 @@ func filterMatchingRe(blockDevices []*blkutils.BlockDevice, nameRegexp, modelReg
 
 	var devices []string
 	for _, blockDevice := range blockDevices {
-		if nameRe != nil && !nameRe.MatchString(blockDevice.Name) {
+		if nameRe != nil && !(nameRe.MatchString(blockDevice.Name) || (nameRe.MatchString(blockDevice.APIName))) {
```

Comment:
No, again, raid controller lists block devices using `lsblk` which only contain kernel names of loop devices. Because user in API provides paths that he knows (`/dev/loops/my-loop-name`), we decorate lsblk output with API to kernel name mapping.